extensions/gcm_message_handler.go

```diff
@@ -287,16 +309,16 @@ func (g *GCMMessageHandler) handleGCMResponse(cm gcm.CCSMessage) error {
 	return nil
 }
 
-func (g *GCMMessageHandler) sendMessage(message interfaces.KafkaMessage) error {
+func (g *GCMMessageHandler) sendMessage(_ context.Context, message interfaces.KafkaMessage) error {
```

Comment:
Why define `context.Context` is the variable is not used?

Reply:
Removed